tests/functional/test_examples.py

```diff
@@ -46,33 +55,32 @@ def test_info():
 
 
 def test_latency():
-    latency = run([executable, './palace-latency.py', WAV], capture_output=True)
+    latency = run([executable, LATENCY, WAV], capture_output=True)
     assert 'Opened' in latency.stdout
     assert 'Playing' in latency.stdout
     assert 'Offset' in latency.stdout
 
 
 def test_reverb():
-    reverbs = run([executable, '/palace-reverb.py', '-p'], capture_output=True)
+    reverbs = run([executable, REVERB, '-p'], capture_output=True)
     assert 'Available reverb preset names:' in reverbs.stdout
     fxs = reverbs.stdout.split('\n')[1:]
     for fx in fxs:
-        reverb = run([executable, '/palace-reverb.py', '-r', fx], capture_output=True)
+        reverb = run([executable, REVERB, '-r', fx], capture_output=True)
         assert 'Opened' in reverb.stdout
         assert 'Playing' in reverb.stdout
         assert fx in reverb.stdout
 
 
 def test_stdec():
-    stdec = run([executable, './palace-stdec.py', WAV], capture_output=True)
+    stdec = run([executable, STDEC, WAV], capture_output=True)
     assert 'Opened' in stdec.stdout
     assert 'Playing' in stdec.stdout
 
 
 def test_tonegen():
-    waveforms = ['sine', 'square', 'sawtooth', 'triangle', 'impulse', 'white-noise']
-    for waveform in waveforms:
-        tonegen = run([executable, '/palace-tonegen.py', '-w', waveform], capture_output=True)
+    for waveform in WAVEFORMS:
+        tonegen = run([executable, TONEGEN, '-w', waveform], capture_output=True)
```

Comment:
the pytest way is to [parametrize](https://docs.pytest.org/en/latest/parametrize.html)